pom.xml

```diff
@@ -38,7 +38,7 @@
     <dependency>
       <groupId>com.klarna</groupId>
       <artifactId>hiverunner</artifactId>
-      <version>4.1.0</version>
+      <version>5.2.2-SNAPSHOT</version>
```

Comment:
Replace this with `5.2.2` when it's available in Maven Central and then I'll approve.